src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```

<img width="800" alt="image" src="https://user-images.githubusercontent.com/55802630/173061549-0f9b7ff6-d4eb-4423-9ab9-9d9fce11b7b2.png">